src/store/modules/ui.ts

```diff
@@ -257,6 +271,11 @@ const getters: GetterTree<IUiState, {}> = {
     return params ? JSON.parse(params) : { field: "originalIndex", type: "asc" };
   },
 
+  walletSearchSortParams(state) {
+    const params = state.walletSearchSortParams || localStorage.getItem("walletSearchSortParams");
+    return params ? JSON.parse(params) : { field: "originalIndex", type: "asc" };
```

Comment:
missed it last time, but `originalIndex` does not exist on the wallet search table (so it will thrown an error in the console if you don't manually sort a column first). It would be better to use `balance` as that's the default by which the API returns the wallets too

Reply:
`balance` should then be `desc` too